CMakeLists.txt

```diff
@@ -70,6 +71,29 @@ string(TOUPPER ${ONEDPL_BACKEND} ONEDPL_BACKEND)
 message(STATUS "Using parallel policies with ${ONEDPL_BACKEND} backend")
 string(TOLOWER ${ONEDPL_BACKEND} ONEDPL_BACKEND)
 
+if (ONEDPL_ENABLE_SIMD)
+    # TODO: remove skipping of Intel® oneAPI DPC++ Compiler once it can handle such options.
+    if (NOT CMAKE_CXX_COMPILER MATCHES ".*dpcpp(-cl)?(.exe)?$")
+        foreach(_simd_flag -fiopenmp-simd /Qiopenmp-simd -fopenmp-simd -qopenmp-simd /Qopenmp-simd -openmp:experimental -openmp-simd)
+            string(MAKE_C_IDENTIFIER ${_simd_flag} FLAG_DISPLAY_NAME)
```

Comment:
Why don't you use just "set" here instead of "string with MAKE_C_IDENTIFIER attribute"?

Reply:
Because some symbols can't be used in variable names. The variable will be shown in a configuration log, so you'll be able to see which options are checked.